config/crd/bases/awx.ansible.com_awxs.yaml

```diff
@@ -112,6 +112,9 @@ spec:
                   description: Port to use for the loadbalancer
                   type: integer
                   default: 80
+                awx_pods_annotations:
```

Comment:
We dont have any other variable names prefixed with `awx`. Please rename this to just `pod_annotations`.